src/Connection/StreamConnection.php

```diff
@@ -123,7 +123,7 @@ public function open() : Greeting
             return $this->greeting = Greeting::unknown();
         }
 
-        $greeting = $this->read(Greeting::SIZE_BYTES, 'Unable to read greeting');
+        $greeting = $this->read(Greeting::SIZE_BYTES, 'Error reading greeting');
```

Comment:
For the sake of consistency with other messages, it should be `Error *when* reading ...`, but I like this variant without "when" more. Maybe update all other messages to this shorter form?

Reply:
Makes sense, I've updated all messages to a shorter form.